features/step_definitions/lint-project.rb

```diff
@@ -0,0 +1,4 @@
+Then(/tuist lints the project and fails/) do
+    _, _, status = Open3.capture3("swift", "run", "tuist", "lint", "project" "--path", @dir)
+    refute(status.success?, "Expected 'tuist lint project' to fail but it didn't")
+  end
```

Comment:
Nitpick: The identation in this file doesn't seem to be right. 
Good idea btw moving it into `lint-project.rb`.